### PR TITLE
IB-3344 Add /multi_media EPs

### PR DESCRIPTION
**Pull request by sam-ludlow**
https://io1dev.atlassian.net/browse/IB-3344

### CODE REVIEW

File: api.swagger.yaml [hunk 1]
```diff
@@ -10012,6 +10171,433 @@ components:
...
+              description: 'Multi media image.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/file-schema'
+            author:
+              description: 'The author of this multi media.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/user-trimmed-schema'
+            meida:
```
Comment (ParryLuke) [inline api.swagger.yaml:10207]: media

File: api.swagger.yaml [hunk 2]
```diff
@@ -10012,6 +10171,433 @@ components:
...
+              description: 'Multi media file.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/file-schema'
+            media_smil:
+              description: 'SMIL file for the Multi media.'
+              type: object
+              allOf:
+                - $ref: '#/components/schemas/file-schema'
+            meida_tta:
```
Comment (ParryLuke) [inline api.swagger.yaml:10217]: media

File: api.swagger.yaml [hunk 3]
```diff
@@ -10012,6 +10171,433 @@ components:
...
+            author:
+              description: 'The author of this multi media.'
+              type: object
+              properties:
+                uuid:
+                  description: 'Invotra UUID of the user.'
+                  type: string
+                  format: uuid
+                  example: '01234567-89ab-cdef-1234-56789abcdef0'
+            meida:
```
Comment (ParryLuke) [inline api.swagger.yaml:10328]: media

File: api.swagger.yaml (hunk 1 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: done

File: api.swagger.yaml (hunk 2 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: done

File: api.swagger.yaml (hunk 3 above)
Comment (sam-ludlow) [reply api.swagger.yaml]: done

**Comment by sam-ludlow** [comment]
Moved changes into https://github.com/invotra/api/pull/292

**Pull request closed without merging**